### PR TITLE
Add new rules and update existing rules - 2nd pass through the list

### PR DESCRIPTION
**Pull request by Softvision-ValentinBandac**
Added 63 new rules for:
researchgate.net, amazon.co.uk, yandex.ru, ya.ru, quora.com, hp.com, dailymail.co.uk, shopify.com, slideshare.net, wiley.com, alibaba.com, cnbc.com, index.hr, protothema.gr, turbopages.org, nrk.no, sapo.pt, blick.ch, yle.fi, amazon.it, pravda.com.ua, iefimerida.gr, svt.se, net.hr, gazzetta.gr, topky.sk, foreca.fi, startlap.hu, olx.pl, blic.rs, digisport.ro, iol.pt, dennikn.sk, ilmatieteenlaitos.fi, newsbomb.gr, telex.hu, shmu.sk, tsn.ua, in.gr, hvg.hu, cmjornal.pt, kinopoisk.ru, telenet.be, sdna.gr, 444.hu, kukaj.io, smhi.se, wykop.pl, publico.pt, seiska.fi, youweekly.gr, ceneo.pl, ojogo.pt, pik.bg, prosport.ro, hnonline.sk, galaxus.de, freemail.hu, polovniautomobili.com, marica.bg, sorozatbarat.club, ceskatelevize.cz, bfmtv.com. 
Removed broken cookie rules for:
sourceforge.net, dr.dk, sme.sk, idokep.hu, kupujemprodajem.com, origo.hu, interia.pl, 24chasa.bg, ekstrabladet.dk, repubblica.it, blitz.bg, ebay-kleinanzeigen.de, fakti.bg, plovdiv24.bg, marktplaats.nl, 3bmeteo.com, seiska.fi, novosti.rs, bazar.bg, etuovi.com, noticiasaominuto.com, telegraf.rs, flashback.org, ampparit.com, meteo.gr, videa.hu, hitta.se, espreso.co.rs

### CODE REVIEW

File: cookie-banner-rules-list.json [hunk 1]
```diff
@@ -1136,11 +1065,11 @@
...
     {
       "click": {
-        "optIn": "button#privacy-cor-wall-accept",
-        "presence": "div.privacy-cor-wall"
+        "optIn": "button#privacy-cp-wall-accept",
+        "presence": "div.privacy-cp-wall"
       },
       "cookies": {},
-      "id": "caf3084b-dfc5-40ca-b66e-a477913efc67",
+      "id": "b90700d6-32a6-44e2-adf4-46acd54089c9",
```
Comment (Trikolon) [inline cookie-banner-rules-list.json:1072]: Is this ID change deliberate? Why is it needed?

Comment (cboozar) [reply cookie-banner-rules-list.json:1072]: +1

**Review by Trikolon — APPROVED** [review]

File: cookie-banner-rules-list.json [hunk 2]
```diff
@@ -1404,16 +1287,12 @@
...
-          {
-            "name": "euconsent-v2",
-            "value": "CPgpnrRPgpnrRFzABBFRBdCgAP_AAH_AAAqIH7Nf_X__b3_n-_79__t0eY1f9_7_v-0zjhfdt-8N2f_X_L8X_2M7vF36pr4KuR4ku3bBIQdtHOncTUmx6olVrTPsbk2Mr7NKJ7Pkmnsbe2dYGH9_n93T_ZKZ7______7________________________-_____9____________wP2AJMNS-AC7EscGSaNKoUQIQrCQ6AUAFFAMLRNYQMrgp2VwEeoIGACE1ARgRAgxBRiwCAAQCAJCIgJADwQCIAiAQAAgBUgIQAEbAILACwMAgAFANCxAigCECQgyOCo5TAgIkWignsrAEou9jTCEMosAKBR_RUYCJQggWBkJCwcxgAA"
-          }
-        ]
+      "click": {
+        "optIn": "button#privacy-cp-wall-accept",
+        "presence": "div.privacy-cp-wall"
       },
-      "id": "4aa16185-4697-4bc4-b0b5-6afecd64c77f",
```
Comment (cboozar) [inline cookie-banner-rules-list.json:1416]: Same question as @Trikolon above, why does this id need to change?

File: cookie-banner-rules-list.json (hunk 1 above)
Comment (Softvision-ValentinBandac) [reply cookie-banner-rules-list.json:1072]: It does not need to change, that is a personal error. Only the CSS selectors needed to be changed.

File: cookie-banner-rules-list.json (hunk 2 above)
Comment (Softvision-ValentinBandac) [reply cookie-banner-rules-list.json:1416]: It does not need to change, that is a personal error. Only the CSS selectors needed to be changed.

**Pull request merged**